engine/apps/integrations/views.py

```diff
@@ -391,3 +391,39 @@ def _process_heartbeat_signal(self, request, alert_receive_channel):
         process_heartbeat_task.apply_async(
             (alert_receive_channel.pk,),
         )
+
+
+class AlertManagerV2View(BrowsableInstructionMixin, AlertChannelDefiningMixin, IntegrationRateLimitMixin, APIView):
+    """
+    AlertManagerV2View consumes alerts from AlertManager. It expects data to be in format of AM webhook receiver.
+    """
+
+    def post(self, request, *args, **kwargs):
+        alert_receive_channel = self.request.alert_receive_channel
+        if not alert_receive_channel.integration == AlertReceiveChannel.INTEGRATION_ALERTMANAGER_V2:
+            return HttpResponseBadRequest(
+                f"This url is for integration with {alert_receive_channel.config.title}."
+                f"Key is for {alert_receive_channel.get_integration_display()}"
+            )
+        alerts = request.data.get("alerts", [])
```

Comment:
consider limiting the maximum number of alerts

Reply:
Actually it is limited when receiving a request, by DATA_UPLOAD_MAX_MEMORY_SIZE, which is set to 1mb. Do we want to add more trimming here?